lib/ngx/balancer_stream.lua

```diff
@@ -0,0 +1,82 @@
+-- Copyright (C) Yichun Zhang (agentzh)
```

Comment:
@splitice It's already too late to introduce the `ngx.http` and ngx.stream` namespace. Also it fragment the API namespace unnecessarily. The hope is that most Lua modules can work automatically in both contexts.